docs/src/examples/components/Toolbar/Types/ToolbarExampleEditor.shorthand.tsx

```diff
@@ -129,7 +131,36 @@ const ToolbarExampleShorthand = () => {
         { key: 'bullets', icon: { name: 'bullets', outline: true } },
         { key: 'number-list', icon: { name: 'number-list', outline: true } },
         { key: 'divider3', kind: 'divider' },
-        { key: 'more', icon: { name: 'more', outline: true } },
+        {
+          key: 'more',
+          icon: { name: 'more', outline: true },
+          active: moreMenuOpen,
+          menu: {
+            items: [
+              {
+                content: 'Quote',
+                icon: 'quote',
+                onClick: () => alert('... -> Quote'),
```

Comment:
I am not a big fun of using `alert()` in example, can we use `Event Log` instead:
https://stardust-ui.github.io/react/components/portal#types-portal

To get prettier output we can use `CodeSnippet`:

```jsx
import { CodeSnippet } from '@stardust-ui/docs-components'

<CodeSnippet label="Output log"  mode="json" value={log} />
```

Reply:
Added event log, cannot use `CodeSnippet` now - missing `minHeight`, no light variant...
Let's think about creating reusable `Log` component